docs/docs/02-concepts/03-platforms.md

```diff
@@ -0,0 +1,235 @@
+---
+title: Platform
+id: platform
+description: Wing platforms
+keywords: [Wing platform, provisioning engines, multi-cloud]
+---
+
+When working with the Wing programming language, an integral part of the compilation process is the use of platform. In essence, platform specify how and where your application is deployed. They determine both the cloud environment and the provisioning engine that the code will be deployed with.
+
+## Platforms
+
+You can view the list of available builtin platform with the `wing compile --help` command. Here is an example of the output:
+
+```sh
+mywing compile --help         
+Usage: wing compile [options] [entrypoint]
+
+Compiles a Wing program
+
+Arguments:
+  entrypoint                    program .w entrypoint
+
+Options:
+  -t, --platform <platform...>  Target platform provider (builtin: sim, tf-aws, tf-azure, tf-gcp, awscdk) (default: ["sim"])
+  -h, --help                    display help for command
+```
+
+In the Options section above, there are several builtin platform that are shipped with Wing, such as `sim`, `tf-aws`, `tf-azure`, `tf-gcp`, and `awscdk`.
+
+These providers contain a combination of provision engine and cloud environment in their names, we refer to these as the platform models (which is discussed in more detail below). The only exception is `sim`, which is a special platform for testing and simulating applications locally.
+
+### Specifying Multiple Platforms
+
+You may have noticed that the `--platform` option accepts multiple arguments. This means you can specify multiple platforms to compile your application to. For example, if you wanted to compile your application using multiple platforms
+
+```sh
+wing compile app.main.w --platform tf-aws platform-foo platform-bar
```

Comment:
Don't you need to specify `-p` multiple times in this case?

Reply:
Nope, commander variadic commands just allow you to keeep space separated. Which is kinda nice and kinda not because it makes it need to come after the entrypoint file. I think I’d prefer needing multiple -p but I haven’t found a way that works